### PR TITLE
Trace python opentelemetry

### PR DESCRIPTION
**Pull request by aravinsiva**
Migrating python-trace-sample docs to OpenTelemetry. 

note: requirements.txt will be updated with opentelemetry package once new version that contains opentelemetry exporter has been released.

### CODE REVIEW

File: trace/trace-python-sample-opentelemetry/main_test.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline trace/trace-python-sample-opentelemetry/main_test.py:1]: new file, this year 2020 for CR

File: trace/trace-python-sample-opentelemetry/main_test.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    main.app.config['TRACER'] = main.initialize_tracer(project_id)
+    client = main.app.test_client()
+
+    resp = client.get('/index.html')
+    assert resp.status_code == 200
+    assert 'Tracing requests' in resp.data.decode('utf-8')
+
+
+def test_redirect():
+    project_id = os.environ['GCLOUD_PROJECT']
```
Comment (gguuss) [inline trace/trace-python-sample-opentelemetry/main_test.py:32]: I think we prefer `GOOGLE_CLOUD_PROJECT` now

File: trace/trace-python-sample-opentelemetry/main.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+# [END trace_setup_python_quickstart]
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.add_argument(
+        '--project_id', help='Project ID you want to access.', required=True)
```
Comment (gguuss) [inline trace/trace-python-sample-opentelemetry/main.py:71]: Add default from environment variable `GOOGLE_CLOUD_PROJECT`

File: trace/trace-python-sample-opentelemetry/main.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Google LLC
```
Comment (gguuss) [inline trace/trace-python-sample-opentelemetry/main.py:1]: new file, update year 2020

File: trace/trace-python-sample-opentelemetry/main.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+@app.route('/', methods=['GET'])
+def root():
+    return redirect(url_for('index'))
+
+
+# [START trace_setup_python_quickstart]
+@app.route('/index.html', methods=['GET'])
+def index():
+    tracer = app.config['TRACER']
+    with tracer.start_as_current_span("index"):
```
Comment (gguuss) [inline trace/trace-python-sample-opentelemetry/main.py:54]: in other strings, you're preferring single-quotes (as do I, thank you), be consistent

**Comment by busunkim96** [comment]
@aravinsiva Please resolve the lint errors:

```
pip install flake8 flake8-import-order
flake8 --show-source --builtin=gettext --max-complexity=20 --import-order-style=google --exclude=.nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py --ignore=E121,E123,E126,E203,E226,E24,E266,E501,E704,W503,W504,I202 --max-line-length=88 --application-import-names main,noxfile,main_test,.nox .
./main.py:18:1: I100 Import statements are in the wrong order. 'import os' should be before 'import time'
import os
^
./main.py:23:1: F401 'opentelemetry.propagators' imported but unused
from opentelemetry import trace, propagators
^
./main.py:23:1: I101 Imported names are in the wrong order. Should be propagators, trace
from opentelemetry import trace, propagators
^
Command flake8 --show-source --builtin=gettext --max-complexity=20 --import-order-style=google --exclude=.nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py --ignore=E121,E123,E126,E203,E226,E24,E266,E501,E704,W503,W504,I202 --max-line-length=88 --application-import-names main,noxfile,main_test,.nox . failed with exit code 1
Session lint failed.
```

**Pull request merged**